pkg/trace/api/otlp.go

```diff
@@ -363,39 +357,85 @@ func convertSpan(rattr map[string]string, lib pdata.InstrumentationLibrary, in p
 	if in.Events().Len() > 0 {
 		span.Meta["events"] = marshalEvents(in.Events())
 	}
+	var ctags strings.Builder // collect container tags from attributes
 	in.Attributes().Range(func(k string, v pdata.AttributeValue) bool {
 		switch v.Type() {
 		case pdata.AttributeValueTypeDouble:
 			span.Metrics[k] = v.DoubleVal()
 		case pdata.AttributeValueTypeInt:
 			span.Metrics[k] = float64(v.IntVal())
 		default:
-			span.Meta[k] = v.AsString()
+			switch k {
+			case "operation.name":
+				span.Name = v.AsString()
+			case "service.name":
+				span.Service = v.AsString()
+			case "resource.name":
```

Comment:
I can't find this on the specification (nor `span.type` or `operation.name`). Where do these come from? An SDK?